src/pages/news/2024-04-15-a-tale-of-two-bugs.md

```diff
@@ -1,17 +1,26 @@
 ---
 template: news-item
-title: A tale of two bugs
-description: The FTX exploit attempted redemption of tBTC revealed two bugs.
+title: "A Tale of Two Bugs: Uncovering Vulnerabilities in tBTC Amid the FTX Exploit"
```

Comment:
All the subtitles directly in the interface are hard to read... death by a thousand colons.

Can we achieve what you want here (indexing benefit) without making the pieces so hard to read?

Reply:
the repetition of phrases in H1 text is the key. can we shorten instead? kw to retain is "tBTC" and IMO "FTX exploit"

What about "A Tale of Two Bugs: tBTC and the FTX Exploit"

Reply:
That's fine.